src/nvim/insexpand.c

```diff
@@ -2279,14 +2279,13 @@ void set_buflocal_cfu_callback(buf_T *buf)
 /// Invoked when the 'omnifunc' option is set. The option value can be a
 /// name of a function (string), or function(<name>) or funcref(<name>) or a
 /// lambda expression.
-int set_omnifunc_option(void)
+void set_omnifunc_option(char **errmsg)
```

Comment:
if we are going to refactor, we could go all the way and pass the buffer here.

Reply:
Yep, I accidentally missed this one.